partner_label/models/res_company.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2017-2019 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    partner_labels_width = fields.Float(
```

Comment:
I don't think you add required for float, because it can save as 0.0 so It would be nice if you can give an api.constrains to avoid partner_labels_width and  partner_labels_height with 0.0 value.